src/Mod/Sketcher/Gui/SketcherToolDefaultWidget.cpp

```diff
@@ -94,27 +94,32 @@ bool SketcherToolDefaultWidget::eventFilter(QObject* object, QEvent* event)
 void SketcherToolDefaultWidget::parameterOne_valueChanged(double val)
 {
     Q_UNUSED(val);
-    QMetaObject::invokeMethod(ui->parameterTwo, "setFocus", Qt::QueuedConnection);
+    isSet[0] = 1;

```

Comment:
Do not change this now.  

But please do not use "1" to assign to a boolean, use true instead. It makes the code much more readable.

I see you got the idea of iterating arrays for a generic default widget.